cog_worker/worker.py

```diff
@@ -261,13 +264,15 @@ def _read_COG(
     crs: Union[str, int, Proj],
     width: int,
     height: int,
+    __retries: int = 6,
```

Comment:
hmm I find a bit counter-intuitive to have a private variable exposed as function parameter. Since `_read_COG` is already "private" does it make sense to have `__retries` instead of `retries` if it is intended to be changed by the user in case the reading fails?